backend/app/models/group.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Group < ApplicationRecord
+  belongs_to :user
+  validates :name, length: { minimum: 4, maximum: 20 }
```

Comment:
Why do we want to have `maximum: 20` limit there? Let's not limit users without specific reasons

Reply:
I thought that will be owfull if name of group will something like that: **"Ruby On Rails Group with my friends and my grand ma (granny)"**